packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/Controls.tsx

```diff
@@ -128,7 +153,7 @@ export const LocationDetailViewControls = (): React.JSX.Element => {
   });
 
   return (
-    <>
+    <DropZoneElement handleDroppedFiles={(files) => handleDroppedFiles(files)}>
```

Comment:
As noted in the _UploadControls.tsx_, drag and drop should be restricted to the `Table`

Reply:
Moved to `Table`